packages/@okta/vuepress-site/docs/reference/api/policy/index.md

```diff
@@ -762,6 +762,8 @@ Different Policy types control settings for different operations. All Policy typ
 * [OAuth Authorization Policy](/docs/reference/api/authorization-servers/#policy-object)
 * [Authentication Policy](#authentication-policy) <ApiLifecycle access="ie" /><br>
 * [Profile Enrollment Policy](#profile-enrollment-policy) <ApiLifecycle access="ie" /><br>
+* [Entity Risk Policy](#entity-risk-policy) <ApiLifecycle access="ie" /><br>
```

Comment:
```suggestion
* [Entity risk policy](#entity-risk-policy) <ApiLifecycle access="ie" /><br>
```

Reply:
The prod docs use sentence case for this; may not be applicable for API. I'll defer to you